query-service-impl/build.gradle.kts

```diff
@@ -34,29 +19,52 @@ dependencies {
     implementation("com.fasterxml.jackson.core:jackson-databind:2.14.2") {
       because("Multiple vulnerabilities")
     }
-    implementation("com.101tec:zkclient:0.11") {
-      because("Multiple vulnerabilities")
+    implementation("org.apache.calcite:calcite-core:1.34.0") {
+      because("CVE-2022-39135")
+    }
+    implementation("org.apache.calcite:calcite-babel:1.34.0") {
+      because("CVE-2022-39135")
+    }
+    implementation("org.apache.avro:avro:1.11.1") {
```

Comment:
There's a number of high-risk upgrades in here. Avro, snakeyaml major version etc. Assuming the direct dep hasn't upgraded, make sure to test at runtime thoroughly if you haven't already

Reply:
will verify them

Reply:
Validated on saas-dev-sanbox. Query service came up fine. No error in logs when UI is accessed. UI was loading up fine as well.